Tests/AirbnbSwiftFormatToolTests/AirbnbSwiftFormatToolTests.swift

```diff
@@ -74,26 +74,63 @@ final class AirbnbSwiftFormatToolTest: XCTestCase {
 
     let error = runFormatTool(
       with: MockCommands(
-        swiftFormat: {
+        swiftFormat: { command in
           ranSwiftFormat = true
 
-          // When autocorrecting SwiftFormat returns EXIT_SUCCESS
-          // even if there were violations that were fixed
-          return EXIT_SUCCESS
+          // When autocorrecting, SwiftFormat returns EXIT_SUCCESS even if there were
+          // violations that were fixed, unless you pass in the --strict option.
+          if command.arguments.contains("--strict") {
+            return SwiftFormatExitCode.lintFailure
+          } else {
+            return EXIT_SUCCESS
+          }
         },
-        swiftLint: {
+        swiftLint: { _ in
           ranSwiftLint = true
           return SwiftLintExitCode.lintFailure
         },
-        swiftLintAutocorrect: {
+        swiftLintAutocorrect: { _ in
           ranSwiftLintAutocorrect = true
 
           // When autocorrecting SwiftLint returns EXIT_SUCCESS
           // even if there were violations that were fixed
           return EXIT_SUCCESS
         }))
 
-    XCTAssertEqual(error as? ExitCode, ExitCode(SwiftFormatExitCode.lintFailure))
+    XCTAssertEqual(error as? ExitCode, .failure)
```

Comment:
Is this the definition of `ExitCode`? https://apple.github.io/swift-argument-parser/documentation/argumentparser/exitcode/issuccess

If so, I feel like it may be better to check `isSuccess` than test against `.failure`. Will this check assertion be satisfied for any nonzero failing exit code?